src/utils.js

```diff
@@ -200,32 +220,167 @@ export const getAgeData = (birthMonth, birthYear, age, ageMonths) => {
 
 /**
  * Return grade after min/max filtering and accounting for string values.
- * 
+ *
  * @function getGrade
- * 
+ *
  * @param {string|number|null} inputGrade - The input grade.
  * @param {number} gradeMin - The minimum grade. Default is 0 (for Kindergarten).
  * @param {number} gradeMax - The maximum grade. Default is 12.
- * 
+ *
  * @returns {number} numeric grade
  */
 export const getGrade = (inputGrade, gradeMin = 0, gradeMax = 12) => {
-  const parsedGrade = Number(inputGrade)
+  const parsedGrade = Number(inputGrade);
   let grade;
 
-  if (isNaN(parsedGrade)) {
+  if (Number.isNaN(parsedGrade)) {
     // Assume grade is K, TK, or PK
-    grade = gradeMin
+    grade = gradeMin;
   } else if (parsedGrade < gradeMin) {
     grade = gradeMin;
   } else if (parsedGrade > gradeMax) {
     grade = gradeMax;
   } else {
     // grade is within range and is a number
-    grade = parsedGrade
+    grade = parsedGrade;
   }
 
   return grade;
 };
- 
 
+/**
+ * Return grade after min/max filtering and accounting for string values.
+ *
+ * @const median
+ *
+ * @param {array} array - Array of response times.
+ *
+ * @returns {number} median of array of response times
+ */
+export const median = (array) => {
+  array.sort((a, b) => b - a);
+  const { length } = array;
+  if (length % 2 === 0) {
+    return (array[length / 2] + array[length / 2 - 1]) / 2;
+  }
+  return array[Math.floor(length / 2)];
+};
+
+/**
+ * Returns a function that evaluates the reliability of a run based on the following criteria:
+ *
+ * @param {number} RESPONSE_TIME_LOW_THRESHOLD - The minimum acceptable response time threshold in MS.
+ * @param {number} RESPONSE_TIME_HIGH_THRESHOLD - The maximum acceptable response time threshold in MS.
+ * @param {number} RESPONSE_SIMILARITY_THRESHOLD - The maximum acceptable response threshold as a number of identical responses
+ * @param {number} ACCURACY_THRESHOLD - The minimum acceptable accuracy threshold.
+ * @param {array} ignoredReliabilityFlags - An array of flags that should be ignored when evaluating reliability.
+ * @returns {function} baseValidityEvaluator - A function that evaluates the reliability of a run.
+ */
+export function CreateEvaluateValidity({
```

Comment:
Suggest camelCase for functions:

```suggestion
export function createEvaluateValidity({
```